Will Heikoff Documentation

```diff
@@ -191,8 +191,8 @@ Hid Column M; Manufacturing Employment is unnecessary for current project
 Created new excel file named “California WAC.xslx” containing only Column A (w_geocode) and Column B (Total Employment)
 Saved file as a CSV file for expected easier use in ArcMap when joining data to “CA_block_2000.shp” in order to have entire state of 
 	California’s Block data joined/merged with the total employment for all available blocks
-The block IDs in CA_Block_2000.shp contains 15 characters, an additional 0 at the beginning to show CAs State Code as 06, one more 
-	than in California WAC.csv Blocks ID of which showcases CAs State Code as 6
+The block IDs in CA_Block_2000.shp contains 15 characters, an additional 0 at the beginning to show CAs State Code as 06 and is in a
+	"double" format, one more than in California WAC.csv Blocks ID of which showcases CAs State Code as 6 which is in a "Single" format
```

Comment:
Let's be a little more precise about this. "double" is a specific type of numeric variable; the type has to do with how much data it can store and how precise it can be. This may be where our problem is coming from--perhaps the "double" type is not enough to handle all 14 digits. I don't think there is any such thing as "single." The other variable is a character or text variable (I don't remember which term ArcMap uses). So the main distinction between these two variables is that one is numeric and the other text. There generally aren't a lot of types of text variables, whereas there ARE a bunch of types of numeric variables.